mantidproject/mantid
# Add doctest for ElasticWindowMultiple

Fixes #12723.

To test:
- Review test
- See that it works (knowing the state of the Windows build you probably have to do this locally)
